test/Tester/TestStreamProviders/FailureInjectionStreamProvider.cs

```diff
@@ -0,0 +1,69 @@
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation.Language;
```

Comment:
be careful with including namespaces that are not really used. This is currently harmless in .NET Framework, but it's breaking when we multi target to .NET Core (which we are doing in other branches). This is why we recently removed all superfluous using statements in #1985. We should be careful in new code not to add more.